src/styles/rewiews.scss

```diff
@@ -0,0 +1,16 @@
+.rewiew {
```

Comment:
The class name 'rewiew' is misspelled. It should be 'review' to ensure that the styles are applied correctly.